src/Sdk/Coverage/Coverage.ts

```diff
@@ -21,7 +22,7 @@ export default class CoverageSdk {
     async list(options: CoverageSearchOptions = {}): Promise<CoverageListResponse> {
         const { jsonQuery, page, pageSize, sortOrder } = options;
         const response = await this.apiClient.get<CoverageListResponse>(
-            this.apiClient.buildUrl(this.sdkUrl),
+            buildUrl(this.apiClient.baseUrl, this.sdkUrl),
```

Comment:
Would it make sense if `ApiClient` were operating over relative URLs? Then you don't need to expose this detail at all. Something like this:

```ts
const { coverage } = await apiClient.get(`/v2/coverage/${coverageId}`);
```

Reply:
Good suggestion. Changed with fb42885fc15c8f794ef359fc7d1cdf96c7050146.